overlay.d/07el9/etc/ssh/sshd_config.d/41-rhcos-defaults.conf

```diff
@@ -0,0 +1,6 @@
+# Disable PasswordAuthentication in SSH
+PasswordAuthentication no
+# Disable root login because don't do that
+PermitRootLogin no
```

Comment:
I think this would then again re-break the change made for tmt in

https://github.com/teemtee/tmt/commit/cd9d3176da1355fb99cd9a085498a7b23de0ca08

Given the MCO doesn't support setting a password for `root`, I think we can safely drop this.

(That said, the tmt change should have been using a drop-in too...)

Reply:
👍🏻

Reply:
This would allow logging in as root with a key, though? Seems like we should keep `PermitRootLogin no` if we're keeping the other stuff and fix tmt.

Reply:
Looks like this hack https://github.com/teemtee/tmt/commit/cd9d3176da1355fb99cd9a085498a7b23de0ca08 will not work for RHCOS 9 anyway so it will need a change.